docs/source/install/linux_mac.rst

```diff
@@ -11,16 +11,6 @@
     3) :code:`pip install --upgrade pip`
     4) :code:`pip install -r ./QuNetSim/requirements.txt`
 
-
-.. note::
```

Comment:
I think instead of deleting this, we should move it to the backend documentation. When I was installing ProjectQ on my computer, I faced this issue and it took a lot of time to overcome. It could potentially save someone time if they want to use the ProjectQ backend so I think it's good to be somewhere.

Reply:
Makes sense. I'll move it this weekend and update the PR.

Reply:
Message added to backend and only slightly modified.